src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$main-size: 100px;
+$main-gap: 1px;
```

Comment:
The variables `$main-size` and `$main-gap` should follow the kebab-case naming convention for consistency with CSS code style rules. Consider renaming to `$main-size` to `$main-size` and `$main-gap` to `$main-gap`.